test/models/short_link_test.exs

```diff
@@ -1,5 +1,5 @@
-defmodule Ask.ShortLinkTest do
-  use Ask.ModelCase
+defmodule AskWeb.ShortLinkTest do
+  use Ask.DataCase
```

Comment:
What is this renaming for?

Reply:
This is what Phoenix uses from now on :shrug:

Reply:
I think it goes with the rest of the refactoring: Phoenix doesn't seem to want models anymore, but data stores, abstracted through contexts. It may also be to avoid thinking the ModelCase is meant to test models, when it's meant to help testing modules _with_ models (I need data)?